Enhanced AI/ML resources with new links and references.

Someshdiwan opened the pull request:
- Updated repository with practical experience, AI/ML roles, and problem-solving resources.

- Added new section in repositorie for Generative Adversarial Networks (GANs) and Artificial Neural Networks (ANNs) implementations.

- Included Convolutional Neural Network (CNN) implementation repository for better hands-on learning.

- Updated the resource list with a blog on learning logic gates (AND, OR, XOR) using Perceptron's.

- Improved the roadmap and resource sections to make learning AI/ML easier and more comprehensive.

Someshdiwan committed:
Enhanced AI/ML resources with new links and references.

- Updated repository with practical experience, AI/ML roles, and problem-solving resources.
- Added new GitHub repositories for Generative Adversarial Networks (GANs) and Artificial Neural Networks (ANNs) implementations.

- Included Convolutional Neural Network (CNN) implementation repository for better hands-on learning.

- Updated the resource list with a blog on learning logic gates (AND, OR, XOR) using Perceptrons.

- Improved the roadmap and resource sections to make learning AI/ML easier and more comprehensive.

Someshdiwan commented:
@armankhondker  i forked your repo—great work! Looking forward to exploring/contributing. 🚀

armankhondker commented on README.md:
```diff
@@ -62,6 +65,7 @@ This repository contains free resources and a roadmap to learn Machine Learning
    - [Google Machine Learning Crash Course](https://developers.google.com/machine-learning/crash-course) 
    - [Machine Learning by Andrew Ng](https://www.coursera.org/learn/machine-learning)
    - [Read Hundred-Page ML Book](http://ema.cri-info.cm/wp-content/uploads/2019/07/2019BurkovTheHundred-pageMachineLearning.pdf)
+   - [Learn AI](https://www.lonelyoctopus.com/)
```
Hi, i'd prefer to leave off paid resources like this.

Someshdiwan replied:
Ok Sir @armankhondker

armankhondker commented on README.md:
```diff
@@ -87,6 +91,7 @@ This repository contains free resources and a roadmap to learn Machine Learning
 8. Prepare for AI/ML Job Interviews
    - [Introduction to Machine Learning Interviews](https://huyenchip.com/ml-interviews-book/)
    - [Designing Machine Learning Systems](https://www.oreilly.com/library/view/designing-machine-learning/9781098107956/)
+   - [AND, OR, XOR Logic Gates Blog](https://www.enjoyalgorithms.com/blog/design-perceptron-to-learn-and-or-xor-gates)
```
dont feel this fits into this section

Someshdiwan replied:
I’ll create a new section specifically for external learning? @armankhondker Sir

armankhondker replied:
Id just remove it

Someshdiwan committed:
Done

Move AND OR XOR Logic Gate section to AI/ML Key concepts.

Removed Paid Resources.

Someshdiwan reviewed (commented):
@armankhondker Sir, could you please review and merge the changes now?

Someshdiwan commented:
@armankhondker  Would it be permissible to include some PDFs and text files in the project? Is that something i can do?

armankhondker commented:
just realized the repoes are from your github, ill leave them off for now as I'm using reputable resources online. Nice job though, keep up the learning!
